notebooks/official/generative_ai/text_embedding_api_cloud_next_new_models.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
<p>Please following the notebook template https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb to support Colab installation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>